src/main/groovy/io/gatling/gradle/RecoverEnterprisePluginException.groovy

```diff
@@ -22,8 +22,13 @@ class RecoverEnterprisePluginException {
         } catch (UnsupportedJavaVersionException e) {
             throwTaskExecutionException(f.getThisObject(), """
                 |${e.getMessage()}
-                |In order to target the supported Java bytecode version, please configure targetCompatibility to Java ${e.supportedVersion} in your build configuration. (see https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)
-                |Or, reported class may come from your project dependencies, published targeting Java ${e.version}.
+                |In order to target the supported Java version, please use the following Gradle setting (requires Gradle 6.6 or later):
+                |compileJava {
+                |    options.release = ${e.supportedVersion}
+                |}
+                |See also the Gradle documentation: https://docs.gradle.org/current/userguide/building_java_projects.html#sec:java_cross_compilation
+                |Another solution is to configure a Java toolchain to use Java ${e.supportedVersion}; see https://docs.gradle.org/current/userguide/toolchains.html
```

Comment:
I wanted to mention the toolchain simply because it seems encouraged to use it from reading the Gradle docs; but it does involve installing the corresponding JDKs, which is more work for the user, so I only mentioned it as a second option.